library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
Oh, right!

So, before we mentioned that were planning to update the `0.1.0` tag to point the final release as soon as it gets there (and in the meantime, point to the `rc`).

Since "retagging" seems odd, we thought we could instead only have latest, and tag `0.1.0` as soon as it's official.

This is a one off: In the future we won't put `rc`s in the library.